pkg/server/server.go

```diff
@@ -92,6 +92,12 @@ var (
 		NativeHistogramMaxBucketNumber:  100,
 		NativeHistogramMinResetDuration: time.Hour,
 	}, []string{"grpc_service", "grpc_method", "datastore_query_count", "dispatch_count", "consistency"})
+
+	throttledRequestCounter = promauto.NewCounterVec(prometheus.CounterOpts{
+		Namespace: build.ProjectName,
+		Name:      "throttled_requests_count",
+		Help:      "The total number of requests that have been throttled.",
+	}, []string{"grpc_method"})
```

Comment:
```suggestion
	}, []string{"grpc_service", "grpc_method"})
```

Reply:
What does this achieve?

Reply:
Make it consistent with the other metrics